test/e2e/scenario/rum/resources.scenario.ts

```diff
@@ -194,8 +194,47 @@ describe('rum resources', () => {
         expect(resourceEvent).toBeTruthy()
         expect(resourceEvent?.resource.status_code).toBe(0)
       })
+
+    createTest('aborting a request stops the response download')
+      .withLogs({ forwardErrorsToLogs: true })
+      .withRum()
+      .run(async ({ servers }) => {
+        await browserExecuteAsync((done) => {
+          const controller = new AbortController()
+          const signal = controller.signal
+
+          fetch('/large-response', { signal }).then(() => {
+            controller.abort()
+            done(undefined)
+          }, console.log)
+        })
+
+        await flushEvents()
+        expect(servers.base.app.getLargeResponseWroteSize()).toBeLessThan(LARGE_RESPONSE_MIN_BYTE_SIZE)
+      })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** is this behavior related to SDK code?